jscomps/UserAgentOverrideHelper.js

```diff
@@ -96,7 +96,7 @@ var UserAgent = {
                              "http-on-modify-request");
     Services.prefs.addObserver(PREF_OVERRIDE, this, false);
     this._customUA = this.getCustomUserAgent();
-    ChromeUtils.import("resource://gre/modules/UserAgentOverrides.jsm");
+    var { UserAgentOverrides } = ChromeUtils.import("chrome://embedlite/content/UserAgentOverrides.jsm");
```

Comment:
I wonder if I need to hoist this import out of the function to avoid the var going out of scope and being collected?

Reply:
Hoisted now.